dashboard/test/dsl/script_dsl_test.rb

```diff
@@ -659,7 +659,7 @@ def setup
     script_level = create :script_level, levels: [level], lesson: lesson, script: script
     script_text = ScriptDSL.serialize_to_string(script_level.script)
     expected = <<~SCRIPT
-      lesson_group 'required', display_name: 'Overview'
+      lesson_group 'required', display_name: 'translation missing: en-US.data.script.name.#{script.name}.lesson_groups.#{lesson_group.key}.display_name'
```

Comment:
I had forgotten that these PL flex categories get used across scripts. I think this is ok, but here are the specific concerns that come to mind:

1. curriculum writers can now add their own lesson groups, it shouldn't be a problem for them to have to create a new lesson group in each new plc script.

2. crowdin is supposed to be able to recognize when the same English string (display name in this case) has already been translated. @Hamms can you confirm this? this should prevent us from re-translating these strings every time we create a new PL course.

3. my recollection is that the flex categories of a script somehow end up as the module_type on PLC learning modules, which have to be one of these values: https://github.com/code-dot-org/code-dot-org/blob/656517757106252cf864c71894448e947d2cdd64/dashboard/app/models/plc/learning_module.rb#L40 I'm not sure exactly how this will play out, but it's possible you will have to change how this works to get tests passing when you go to move off of flex categories.

Reply:
Thanks Dave for pointing this out again. It is on my radar as something I need to figure out how to deal with. I was considering doing something where if its a PLC scripts then we reserve those module type keys so that you couldn't somehow make a new key-display_name pair for those reserved keys.

Also I confirmed with Elijah that if a translation already exists Crowdin is smart about applying it to new instances of that same string.

Reply:
hah, thank you for understanding, I had forgotten pointing this out before :-) I think a special case for the PLC course types sounds great.

Reply:
note you can add a default to the `I18n.t` call in `localized_display_name`, for those situations where the string hasn't yet been persisted to scripts.en.yml

Reply:
Dealt with in this PR: https://github.com/code-dot-org/code-dot-org/pull/34348